README.md

```diff
@@ -148,11 +148,27 @@ represents Brazilian Portuguese.
 
 ## Repository Structure
 
+### Branches and Snap Packages
+
+Active development for all snaps built from this monorepo takes place on the
+`main` branch. To continue supporting Ubuntu releases with bug fixes dedicated
+branches (e.g. `ubuntu/24.04`) are created once development on features for an
+upcomiingrelease begins.
```

Comment:
upcoming release

Reply:
I thought I spell-checked this :D thanks